### PR TITLE
Newform appointment

### PR DESCRIPTION
**Pull request by raquelgraca**
Set the message error in the field "end_time" and set new position for the error base

### CODE REVIEW

File: app/models/play_space.rb [hunk 1]
```diff
@@ -14,7 +14,7 @@ class PlaySpace < ApplicationRecord
 
   def photos_presence_format
     if photos.attached? == false
-      errors.add(:photos, "Photos required.")
+      errors.add(:photos, "required.")
```
Comment (diego-unternaehrer) [inline app/models/play_space.rb]: Why did you change that? Wouldn't "Photos required." be a nicer error message?

Comment (raquelgraca) [reply app/models/play_space.rb]: I changed because in the way before the message was > Photos Photos required. 
Rails is showing the name of the input as part of the message.

Now, the message is "Photos required"
